### ISSUE TITLE
Question: Downloading files addition

### ISSUE THREAD

**Issue by christophertull** [21Buttons/pychromeless#12]
Hi there. First off, this utility is amazing! Like the ultimate prepackaged webscraping environment. So thanks for making it public!

I noticed while using the package that headless chromium currently has some issues downloading files, but there are [workarounds](https://bugs.chromium.org/p/chromium/issues/detail?id=696481#c157) online. I have added these to my own fork, but wanted to ask if you would be interested in a PR to add these workarounds to this repo?

Maybe something like a `download` flag in the `WebDriverWrapper` that sets the preferences for downloading and adds the headless workaround?

**Comment by davesgonechina** [comment]
@christophertull I'm trying to get the same workaround to work properly, have you been successful and if so can you share how you did it?

**Comment by christophertull** [comment]
Hi @davesgonechina I went ahead and opened a PR with my changes. Hopefully that code works for you!

https://github.com/21Buttons/pychromeless/pull/13

The step that got it to finally work for me was that my download link was opening the download in another tab, so I found a workaround that first replaces all of the `target='_blank'` with `target='_self'` so that the download opens in the same tab instead.
